content/blog/2020/06/2020-06-08-hackfest-docs-results.adoc

```diff
@@ -0,0 +1,113 @@
+---
+layout: post
+title: "Jenkins User Experience Hackfest Documentation Results"
+tags:
+- ux
+- documentation
+- community
+authors:
+- markewaite
+- tracymiranda
+links:
+  gitter: jenkinsci/docs
+opengraph:
+  image: /images/post-images/2020-06-08-docs-from-hackfest/docs-from-hackfest-opengraph.png
+---
+
+> Documentation is not glamorous, but is is goodness.
+> -- Thomas Otter, https://www.otteradvisory.com/2020/01/documentation-and-safety/
+
+Jenkins technical documentation is an important part of our project as it is key to using Jenkins well.
+Good documentation guides users and encourages good implementation choices.
+It is a crucial part of the user experience.
+
+In the recent Jenkins UI/UX hackfest, documentation was a specific track to improve the Jenkins user experience.
+We received many improvements from experienced Jenkins contributors and newcomers alike.
+Contributors from all around the world submitted pull requests for documentation on installing, managing, administering, and operating Jenkins.
+
+image:/images/post-images/2020-06-08-docs-from-hackfest/contributions-by-country.png[Contributors to Docs by country]
+
+== Documentation migration from Wiki
+
+The Jenkins Wiki pages have collected 15 years of experience and wisdom for Jenkins users.
+However, that experience and wisdom is intermixed with inaccurate, incomplete, and outdated information.
+
+The Jenkins Wiki migration project identified the 50 most accessed pages on the Jenkins wiki and created GitHub issues to track the migration of those pages to www.jenkins.io.
+This was our first large scale experiment using GitHub issues for documentation.
+The results have been overwhelmingly positive.
+Hackfest contributors added new sections to many documentation chapters, including:
+
+* link:/doc/book/using/[Using Jenkins]
+* link:/doc/book/pipeline/[Pipeline]
+* link:/doc/book/managing/[Managing Jenkins]
+* link:/doc/book/system-administration/[System Administration]
+
+The Hackfest closed 19 of the wiki migration issues.
+Work is in progress on an additional 25 wiki migration issues.
+We've made great progress and look forward to even better results in the future.
+New contributors used the "good first issue" label very effectively.
+We started the Hackfest with most of the 25 "good first issues" unassigned and completed the Hackfest with 14 closed and 10 others in progress.
+We'll provide more "good first issues" as we use the Jenkins Wiki migration to welcome new documentation contributors.
+
+image:/images/post-images/2020-06-08-docs-from-hackfest/sample-pages.png[Sample Hackfest documentation pages, width="80%"]
```

Comment:
This one looks to be over-compressed. The text is undereadable on the current image

Reply:
Thanks.  I was too aggressive in image optimization with that one.  Fixed in 752d56de0a3cb44075ff965d2c13842554027c9a